### ISSUE TITLE
Add developer note

### ISSUE THREAD

**Issue by mbukosky** [falcordotnet/falcor.net#8]
Hi,

I really like the project you have put together. I just wanted to share with you a little trouble I ran into when using this solution. I had a difficult time getting the simple helloWorld.json to run because of a dumb web.config setting.

The "standard" webconfig setting is

```
<handlers>
	<add name="ExtensionlessUrlHandler-Integrated-4.0" path="*." verb="*" type="System.Web.Handlers.TransferRequestHandler" preCondition="integratedMode,runtimeVersionv4.0" />
</handlers>
```

but it seems this doesn't work with the routering. It needs to have an extra "*"

```
<handlers>
	<add name="ExtensionlessUrlHandler-Integrated-4.0" path="*.*" verb="*" type="System.Web.Handlers.TransferRequestHandler" preCondition="integratedMode,runtimeVersionv4.0" />
</handlers>
```

Maybe, it would be nice to have a little section in the readme file or something. Just a thought. Hope this helps out someone else.

**Comment by craigsmitham** [comment]
Thanks for the feedback! I remember running into this issue but neglected to document it.

Do you think it would make sense to add a Powershell script to add this to the web.config? I think it probably only applies if one is using IIS ...

**Comment by mbukosky** [comment]
Yeah, I think that PS is overkill in this case. Just a simple, notes or gotcha section would be fine. Devs are smart enough to figure it out.
